app/src/app/[locale]/importer/[id]/validate/cells/SelectCell.tsx

```diff
@@ -38,37 +50,82 @@ const SelectCell = ({
   isRequired,
   availableValues,
   isReadOnly,
+  configKey,
+  importerId,
+  onReloadConfig,
+  allowAddNewValue,
 }: Props) => {
+  const { t } = useTranslation();
   const id = useId();
+  const frontendFetch = useFrontendFetchWithAuth();
   const [value, setValue] = React.useState<string>(initialValue);
+  const [dialogOpen, setDialogOpen] = React.useState<boolean>(false);
+
   React.useEffect(() => {
     setValue(initialValue);
   }, [initialValue]);
+
+  const handleDialogSave = async (value: string) => {
```

Comment:
Eventuell wäre es sauberer, wenn die Cell-Components möglichst wenig Logik enthalten und man das hinzufügen über einen Callback nach außen weitergibt?

Reply:
hatte ich auch überlegt, aber dann war mir die Logik in der ValidationTable component zu kompliziert - ich muss ja dann die value von der SelectCell neu setzten, deswegen hab ich es in der Cell gelassen. Geht aber sicher besser :-)